### ISSUE TITLE
questions regarding roadmap?

### ISSUE THREAD

**Issue by rcon420** [mEldevlp/surfcombat-metamod-cs2#5]
im curious why u don't focus on the feature that are only related to the combat surf?  You could build simple core plugin for surf combat and all the other stuff can be done by other plugins or u can make ur own s plugins that arent build in the core.

- For  Admins Tools / Movment unlocker and other fixes there is cs2fixes: https://github.com/Source2ZE/CS2Fixes
For CS2 Ranking you can use:   https://github.com/Salvatore-Als/cs2-rank or cssharp rank plugins (your road map doesn't really offer anything different).
- !hide is in cs2fixes or with css sharp plugin can be one
- cs2fixes has rtv
- reset score be easy done by adding few lines in cs2fixes or with css sharp plugin
- for event stuff there are so many plugins already done
- for chat player prefix or tab there are already many plugins

tl;dr only things that other plugins don't  offer right now is:

-Red and Blue player models
- Spawn protection 10 seconds
-Purchase of privileges via GameCMS or tg bot

don't take it as insult but if u want normal movement u already need cs2fixes and u  will just waste ur time building so many functions  while u can just use other plugins that are backed by strong active dev team/coder.

**Comment by mEldevlp** [comment]
Of course right you are. But unfortunately this project does not make sense now, because it is easier to create plugins using frameworks (cssharp, swiftly or resoursemod). Maybe I will come back to this project later and finish it as a "classic" surf server plugin

PS I apologize for long time of an answer :(
